pkg/cmd/roachtest/test_runner.go

```diff
@@ -619,10 +619,17 @@ func (r *testRunner) runTest(
 				msg := fmt.Sprintf("The test failed on branch=%s, cloud=%s:\n%s",
 					branch, cloud, output)
 				artifacts := fmt.Sprintf("/%s", t.Name())
+				title := fmt.Sprintf("roachtest: %s failed", t.Name())
+				// Don't disambiguate by branch if the branch is master, to avoid
+				// creating duplicates with issues created prior to when this code
+				// was introduced.
+				if branch != "master" {
+					title = fmt.Sprintf("roachtest: %s failed on %s", t.Name(), branch)
+				}
 
 				req := issues.PostRequest{
```

Comment:
i'm unfamilar with this code -- where do we attempt to find and update an existing issue vs post a new one?

Reply:
it's in `issues` and basically searches github for the title of the desired issue. See my comment in the main thread on how I'm planning to improve it.